roles/pf9-hostagent/files/pf9_get_hostagent.py.j2

```diff
@@ -193,9 +186,9 @@ def get_package_info_from_token(host, token, region, proxy=""):
     return out
 
 
-def get_installer(options):
+def get_installer(options, proxyHost="", proxyPort=""):
     token = get_token_v3(options.endpoint, options.user,
-                         options.pw, options.tenant, options.proxy)
+                         options.pw, options.tenant, proxyHost, proxyPort)
 
     info = get_package_info_from_token(options.endpoint, token, options.region)
```

Comment:
You need to pass proxyHost, proxyPort to the get_package_info_from_token function.

e.g.
info = get_package_info_from_token(options.endpoint, token, options.region, proxyHost, proxyPort)